tox.ini

```diff
@@ -22,12 +22,7 @@ deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
 commands =
-    coverage erase
```

Comment:
I think it would make sense to keep this line since its purpose is to clean the coverage date from the old runs. The coverage data is generated from the "stestr" run (we're overriding the PYTHON to allow coverage to do that).